neovim/msgpack_rpc/event_loop/asyncio.py

```
@@ -111,6 +114,11 @@ def _threadsafe_call(self, fn):
         self._loop.call_soon_threadsafe(fn)
 
     def _setup_signals(self, signals):
+        if os.name == 'nt':
+            # add_signal_handler is not supported in win32
+            self._signals = []
+            return
```

shouldn't this be an error instead of silently failing?